### ISSUE TITLE
How would sessions / login work if the user's frontend is within their client's domain?

### ISSUE THREAD

**Issue by rishabhpoddar** [supertokens/supertokens-core#82]
In this case, the frontend would not have a fixed domain:
- The `websiteDomain` on the frontend and backend would have to change based on the client's domain
   - Since the `websiteDomain` is only used on the backend for forgot password emails, perhaps the user can set the `websitDomain` to be `""` and then use the `getResetPasswordURL` function to generate the link depending on who is logged in?
- Is there a need for `websiteDomain` on the frontend?
- Anything else?

**Comment by NkxxkN** [comment]
Are we talking about multi-tenancy here? 

I think it was added for consistency in `appInfo` originally. So far I haven't used it and I don't think we will need to. We can always do `window.location.origin`.

> websitDomain to be "" and then use the getResetPasswordURL function to generate the link depending on who is logged in?

In that case that would be depending on the domain? Since the user would not be logged in?

**Comment by rishabhpoddar** [comment]
> Are we talking about multi-tenancy here?

No. Here this is a scenario in which someone is providing a dev facing product which embeds in people's website. The embedding requires the end user to sign in / up. Since the end user can have any site (any domain), this problem arrises.

TLDR; This does not seem to be a problem for now since it fits well within our configs. Closing this issue.

> I think it was added for consistency in appInfo originally.

Yes. Correct.

> In that case that would be depending on the domain? Since the user would not be logged in?

For now, we only require the `websiteDomain` to send the forgot password email... In here, we get the user's info in the API to call their callback for them to generate a custom email link depending on the user's domain.
